Shoko.Server/Databases/MySQL.cs

```diff
@@ -805,7 +805,8 @@ public class MySQL : BaseDatabase<MySqlConnection>
         new(129, 35, "ALTER TABLE `TMDB_Movie` CHANGE COLUMN IF EXISTS `EnglishOvervie` `EnglishOverview` TEXT CHARACTER SET UTF8 NOT NULL;"),
         new(129, 36, "UPDATE `TMDB_Image` SET `IsEnabled` = 1;"),
         new(130, 1, MigrateRenamers),
-        new(131, 1, "DELETE FROM RenamerInstance WHERE NAME = 'AAA_WORKINGFILE_TEMP_AAA';")
+        new(131, 1, "DELETE FROM RenamerInstance WHERE NAME = 'AAA_WORKINGFILE_TEMP_AAA';"),
+        new(131, 2, DatabaseFixes.CreateDefaultRenamerConfig)
```

Comment:
can you add a trailing comma now so the next db migration don't have to edit the previous line like you had to above?